src/main/resources/application-gcp-dev.properties

```diff
@@ -0,0 +1,57 @@
+baseUrl=https://pasa-410818.oa.r.appspot.com
+frontendUrl=https://tournament-app-f-dot-pasa-410818.oa.r.appspot.com
+
+# PostgresSql
+spring.datasource.url=jdbc:postgresql://34.118.69.211:5432/postgres
+spring.datasource.username=postgres
+spring.datasource.password=Jlyl}%If6?l(i8hH
+spring.datasource.driver-class-name=org.postgresql.Driver
+
+# This will create table automatically in your database (update/create)
+spring.jpa.hibernate.ddl-auto=create
+
+# Apache Kafka
+spring.kafka.producer.bootstrap-servers=pkc-03vj5.europe-west8.gcp.confluent.cloud:9092
+spring.kafka.consumer.bootstrap-servers=pkc-03vj5.europe-west8.gcp.confluent.cloud:9092
+spring.kafka.consumer.group-id=my-consumer-group
+spring.kafka.consumer.auto-offset-reset=earliest
+spring.kafka.consumer.key-deserializer=org.apache.kafka.common.serialization.StringDeserializer
+spring.kafka.consumer.value-deserializer=org.apache.kafka.common.serialization.StringDeserializer
+spring.kafka.properties.sasl.mechanism=PLAIN
+spring.kafka.bootstrap-servers=pkc-03vj5.europe-west8.gcp.confluent.cloud:9092
+spring.kafka.properties.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username='5YGFD7MNBADEZRLU' password='BM6G6aEatDNTGku6H8qosKOtlZCvDL61OPYuPrKUoHZuvicXsoi2C9K2lUPXiivf';
+spring.kafka.properties.security.protocol=SASL_SSL
+
+
+# LinkedIn OAuth2
+#spring.security.oauth2.client.registration.linkedin.client-id=86xfeosqtas5qu
+#spring.security.oauth2.client.registration.linkedin.client-secret=lGeVodVhSaq0iaD7
+#spring.security.oauth2.client.registration.linkedin.redirect-uri={baseUrl}/login/oauth2/code/linkedin
+#spring.security.oauth2.client.registration.linkedin.authorization-grant-type=authorization_code
+#spring.security.oauth2.client.provider.linkedin.token-uri=https://www.linkedin.com/oauth/v2/accessToken
+#spring.security.oauth2.client.registration.linkedin.scope=profile, email
+#spring.security.oauth2.client.provider.linkedin.authorization-uri=https://www.linkedin.com/oauth/v2/authorization
+#spring.security.oauth2.client.provider.linkedin.user-info-uri=https://api.linkedin.com/v2/me
+
+# GitHub OAuth2
+spring.security.oauth2.client.registration.github.client-id=39f7f796c86a894ac382
+spring.security.oauth2.client.registration.github.client-secret=d516a4dbe51a41b359bf564ab191cdda265df270
+spring.security.oauth2.client.registration.github.redirect-uri=https://pasa-410818.oa.r.appspot.com/login/oauth2/code/github
+spring.security.oauth2.client.provider.github.authorization-uri=https://github.com/login/oauth/authorize
+spring.security.oauth2.client.provider.github.token-uri=https://github.com/login/oauth/access_token
+
+
+# why do we need this???
```

Comment:
In environments using proxies or load balancers, these configurations help maintain security, performance, and proper functioning of application.
Probably Google cloud uses both proxies and load balancers